radicle-cli/src/commands/auth.rs

```diff
@@ -0,0 +1,226 @@
+#![allow(clippy::or_fun_call)]
+use std::ffi::OsString;
+
+use anyhow::anyhow;
+
+use radicle::crypto::ssh;
+use radicle::Profile;
+
+use crate::git;
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+pub const HELP: Help = Help {
+    name: "auth",
+    description: "Manage radicle identities and profiles",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad auth [--init | --active] [<options>...] [<peer-id>]
+
+    A passphrase may be given via the environment variable `RAD_PASSPHRASE` or
+    via the standard input stream if `--stdin` is used. Using one of these
+    methods disables the passphrase prompt.
+
+    If `--init` is used, a name may be given via the `--name` option. Using
+    this disables the input prompt.
+
+Options
+
+    --init                  Initialize a new identity
+    --active                Authenticate with the currently active profile
+    --stdin                 Read passphrase from stdin (default: false)
+    --name <name>           Use given name (default: none)
+    --help                  Print help
+"#,
+};
+
+#[derive(Debug)]
+pub struct Options {
+    pub active: bool,
```

Comment:
Can be removed as well. `rad auth` run when the user has a profile is `rad-auth --active`